scripts/builtin/dist.dml

```diff
@@ -23,7 +23,8 @@
 m_dist = function(Matrix[Double] X) return (Matrix[Double] Y) {
     G = X %*% t(X);
     I = matrix(1, rows = nrow(G), cols = ncol(G));
-    Y = -2 * (G) + t(I %*% diag(diag(G))) + t(diag(diag(G)) %*% I);
+    Y = -2 * (G) + (diag(G) * I) + (I * t(diag(G)));
+#   Y = -2 * (G) + t(I %*% diag(diag(G))) + t(diag(diag(G)) %*% I);
```

Comment:
why change this @OlgaOvcharenko 
I think it does the same but did not verify.

Reply:
Right, the same, just thought it would be simpler